.github/workflows/dispatch.yaml

```diff
@@ -25,12 +25,13 @@ jobs:
           echo "found_image=$found" >> $GITHUB_ENV
       - name: Setup java
         if: ${{ !env.found_image }}
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v3
         with:
+          distribution: 'zulu'
```

Comment:
Dvs samme som i v1 - ref https://github.com/actions/setup-java/blob/main/docs/switching-to-v2.md